### PR TITLE
Master ctauto293

### PR DESCRIPTION
**Pull request by jrdavisllbean**
Please review the WebElements for SecureCheckoutModalMobilePage and SecureCheckoutModalPage classes.

### CODE REVIEW

File: src/main/java/com/llbean/automation/web/pages/desktopNTablet/SecureCheckoutModalPage.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            // fall through, loginFrame could be visible.
+            logger.debug("Catching " + e.getMessage() + ". loginLayer iframe frame is visible.");
+        }
+    }
+
+    void closeMergeBagDialogIfVisible() {
+
+        By itemsSavedFromPreviousVisitDialog = By.cssSelector("div#sb_RemoveDialogDialog div div a[title='OK']");
+        try {
+            getDriver().findElement(itemsSavedFromPreviousVisitDialog).click();
```
Comment (didierllbean) [inline src/main/java/com/llbean/automation/web/pages/desktopNTablet/SecureCheckoutModalPage.java]: Please create a web Element for this using the factory anotation.
Also make sure to add the wait time for the element

File: src/main/java/com/llbean/automation/web/pages/mobile/SecureCheckoutModalMobilePage.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    @FindBy(className = "forgetPwdLink")
+    public WebElement forgotPasswordLink;
+
+    public SecureCheckoutModalMobilePage(WebDriver driver) {
+        super(driver);
+        PageFactory.initElements(driver,this);
+    }
+
+    public void enterLoginEmail(String loginEmail) {
+        WebElement emailAddressTextField = getDriver().findElement(By.id("loginEmail"));
```
Comment (didierllbean) [inline src/main/java/com/llbean/automation/web/pages/mobile/SecureCheckoutModalMobilePage.java]: Add this a webelement using factory annotations, something like this:
@FindBy(id = "loginEmail")
public WebElement login;

File: src/main/java/com/llbean/automation/web/pages/mobile/SecureCheckoutModalMobilePage.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        emailAddressTextField.clear();
+        emailAddressTextField.sendKeys(loginEmail);
+    }
+
+    public void enterPassword(String loginPassword) {
+        logger.info("Entering the login password into the password field.");
+        WebElement passwordTextField = getDriver().findElement(By.id("loginPassword"));
```
Comment (didierllbean) [inline src/main/java/com/llbean/automation/web/pages/mobile/SecureCheckoutModalMobilePage.java]: Add this a webelement using factory annotations.

**Comment by jrdavisllbean** [comment]
CTAUTO-294 Removed all By objects and replaced with factory annotation, added waits per feedback.

**Pull request merged**